tests/unit/Rules/SymbolicLinkTest.php

```diff
@@ -29,56 +33,87 @@ function is_link($link)
 }
 
 /**
- * @group  rule
+ * @group rule
  * @covers \Respect\Validation\Exceptions\SymbolicLinkException
```

Comment:
Because we're testing the exceptions in the PHPT file, there is no need for testing this here.